.circleci/config.yml

```diff
@@ -1,4 +1,6 @@
 version: 2.1 # Adds support for executors, parameterized jobs, etc
+orbs:
+  github-cli: circleci/github-cli@2.1.0
```

Comment:
Orbs have had meh reliability and upgradability in the past. You might be better installing gh straight on the machine as part of devtools.
Or if you already have a gh token just doing a curl is likely enough for what you want here no?

Reply:
done via curl